### PR TITLE
scrolling issues css updates

### PR DESCRIPTION
**Pull request by ghislaineguerin**
stage link : https://apps-stage-4.risevision.com/

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ghislaineguerin looks good to me. 2 things I noticed on staging (when going to smaller resolutions) is the spacing between the CH and the rest of the page jumps (and almost doubles in size) when resizing the page. This is most apparent in the Editor: https://screencast.com/t/sA527s2B3. Wondering if you could take a look. Thanks.

**Comment by ghislaineguerin** [comment]
@alex-deaconu I will take a look.

**Comment by RV-Adrian** [comment]
@ghislaineguerin ^^

**Comment by alex-deaconu** [comment]
@ghislaineguerin the issue I mentioned seems fixed. Thanks.

File: dist/css/rise.css [hunk 1]
```diff
@@ -7564,66 +7564,6 @@ button.close {
   .label-default[href]:hover, .label-default[href]:focus {
     background-color: #fff; }
 
-.navbar {
-  z-index: 1000;
```
Comment (ghislaineguerin) [inline dist/css/rise.css]: @alex-deaconu here's the problem. This needs to be added back.

File: src/scss/ui-components/common-header.scss [hunk 2]
```diff
@@ -1,17 +1,55 @@
+.navbar-default .navbar-header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: 0 8px;
+}
+
 .navbar {
```
Comment (ghislaineguerin) [inline src/scss/ui-components/common-header.scss]: @alex-deaconu add it back here please.

**Pull request merged**